edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Http/controllers/DeviceScopeController.cs

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Http.Controllers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.Amqp.Framing;
+    using Microsoft.Azure.Devices.Edge.Hub.CloudProxy;
+    using Microsoft.Azure.Devices.Edge.Hub.Core;
+    using Microsoft.Azure.Devices.Edge.Hub.Core.Identity;
+    using Microsoft.Azure.Devices.Edge.Hub.Core.Identity.Service;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+    using Org.BouncyCastle.Security;
+
+    public class DeviceScopeController : Controller
+    {
+        static readonly string SupportedContentType = "application/json; charset=utf-8";
+
+        readonly Task<IEdgeHub> edgeHubGetter;
+
+        public DeviceScopeController(Task<IEdgeHub> edgeHub)
+        {
+            this.edgeHubGetter = Preconditions.CheckNotNull(edgeHub, nameof(edgeHub));
+        }
+
+        [HttpPost]
+        [Route("devices/{actorDeviceId}/modules/{actorModuleId}/devicesAndModulesInTargetDeviceScope")]
+        public async Task InvokeAsync([FromRoute] string actorDeviceId, [FromRoute] string actorModuleId, [FromBody] NestedScopeRequest request)
```

Comment:
Nit: Can we name it more specific - like GetDevicesAndModulesInTargetScope or so?